src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,12 @@
+import "./Nav.css";
+
+export default function Nav() {
+  return (
+    <div className="navigation">
```

Comment:
`<nav>` is a nice symantic tag that could be used here!

Reply:
Ah yes, good point!